libsolidity/grammar.txt

```diff
@@ -35,14 +35,20 @@ Continue = 'continue' ';'
 Break = 'break' ';'
 Return = 'return' Expression? ';'
 Throw = 'throw' Expression? ';'
-VariableDefinition = VariableDeclaration ( = Expression )? ';'
+VariableDefinition = VariableDeclaration ( '=' Expression )? ';'
 
 Expression = Assignment | UnaryOperation | BinaryOperation | FunctionCall | NewExpression | IndexAccess |
-			 MemberAccess | PrimaryExpression
+             MemberAccess | PrimaryExpression
 // The expression syntax is actually much more complicated
 Assignment = Expression (AssignmentOp Expression)
+BasicBinaryOperation = '|' | '^' | '&' | '<<' | '>>' | '>>>' | '+' | '-' | '*' | '/' | '%'
+AssignmentOp = BasicBinaryOperation'='
+UnaryOperation = '!' | '~' | '++' | '--' | 'delete'
+BinaryOperation = BasicBinaryOperation | '||' | '&&' | '**' |
```

Comment:
It is probably a good idea to keep it as a reserved keyword, i.e. move it a bit down, and change [isCompareOp()](https://github.com/ethereum/solidity/blob/develop/libsolidity/parsing/Token.h#L270) to use `GreaterThanOrEqual` instead of `in`.